### PR TITLE
Created mobile and web view for landing page

### PR DESCRIPTION
**Pull request by maximeish**
### What this PR does
Add a landing page

### Task to be completed
Create web view for the landing page
Create mobile view for landing page

### Manual testing
Clone the github repository

run npm install

run npm start to start the dev server

### Background context
N/A

### Relevant pivotal tracker stories
https://trello.com/c/4lxg73Ft/45-implement-home-page-design-using-the-new-mockup-design

### Screenshots

> **mobile view**

mobile

> **web view**

web2
web1

### CODE REVIEW

**Review by IRANKUND — CHANGES_REQUESTED** [review]
good job Maxime, but what you can add is to update your scripts so that it will not fail to be deployed to heroku. change start to run "node server" and that script in start it can be in dev

**Comment by maximeish** [comment]
> good job Maxime, but what you can add is to update your scripts so that it will not fail to be deployed to heroku. change start to run "node server" and that script in start it can be in dev

@IRANKUND ok I updated it

**Review by giranezafiacre — COMMENTED** [review]
I like your work @maximeish, but maybe you will increase coverage to make it better

File: src/pages/LandingPage/MapComponent.js [hunk 1]
```diff
@@ -1,26 +1,74 @@
...
+
+
+const MapComponent = (props) => {
+  const samplePosition = {
+    lat: -1.9297648,
+    lng: 29.9871554
+  };
+
+  if (props.origin !== undefined && props.destination !== undefined) {
+    // eslint-disable-next-line no-unused-vars
```
Comment (malaba6) [inline src/pages/LandingPage/MapComponent.js]: Plz don't disable eslint rules

File: src/pages/LandingPage/MapComponent.js [hunk 2]
```diff
@@ -1,26 +1,74 @@
...
+    const [origin, setOrigin] = useState({...props.origin});
+    // eslint-disable-next-line no-unused-vars
```
Comment (malaba6) [inline src/pages/LandingPage/MapComponent.js]: same here

File: src/pages/LandingPage/SearchPanel.js [hunk 3]
```diff
@@ -0,0 +1,257 @@
+import React, { useState } from 'react';
+import directionIcon from 'App/assets/images/direction-icon.svg';
+import backButton from 'App/assets/images/back-button.svg';
+import PropTypes from 'prop-types';
+import originData from 'shared/constants/originBusStops.js';
+import destinationData from 'shared/constants/destBusStops.js';
+import Styled from '@emotion/styled';
+import { TextField } from '@material-ui/core';
+import { Autocomplete } from '@material-ui/lab';
+// import Autocomplete from '@material-ui/lab/Autocoplete';
```
Comment (malaba6) [inline src/pages/LandingPage/SearchPanel.js]: Remove commented out lines

**Review by malaba6 — CHANGES_REQUESTED** [review]

**Review by IRANKUND — APPROVED** [review]

**Pull request merged**